backend/src/Squidex/Areas/IdentityServer/Config/IdentityServerServices.cs

```diff
@@ -127,7 +128,13 @@ public static void AddSquidexIdentityServer(this IServiceCollection services)
 
             options.SuppressXFrameOptionsHeader = identityOptions.SuppressXFrameOptionsHeader;
 
-            options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
+            // Set antiforgery cookie secure policy to always for https
+            var urlsOptions = config.GetSection("urls").Get<UrlOptions>() ?? new ();
```

Comment:
Don't use config here. Try to inject IOptions<UrlOptions> or `IUrlGenerator` (see in the same file), because there should be only one call where the options is binded to the config.

Reply:
Agreed its cleaner without config - see changes